src/explorer/AppSettingsNodes.ts

```diff
@@ -3,30 +3,35 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import * as WebSiteModels from '../../node_modules/azure-arm-website/lib/models';
+import { SubscriptionModels } from 'azure-arm-resource';
+import WebSiteManagementClient = require('azure-arm-website');
+import * as opn from 'opn';
 import * as path from 'path';
 import * as vscode from 'vscode';
-import * as util from '../util';
-import * as opn from 'opn';
+import { TreeItem, TreeItemCollapsibleState } from 'vscode';
+import * as WebSiteModels from '../../node_modules/azure-arm-website/lib/models';
 import { AzureAccountWrapper } from '../AzureAccountWrapper';
-import { NodeBase } from './NodeBase';
+import * as util from '../util';
 import { AppServiceDataProvider } from './AppServiceExplorer';
-import { SubscriptionModels } from 'azure-arm-resource';
-import { TreeItem, TreeItemCollapsibleState } from 'vscode';
-import WebSiteManagementClient = require('azure-arm-website');
+import { NodeBase } from './NodeBase';
 
 export class AppSettingsNode extends NodeBase {
+    public readonly site: WebSiteModels.Site;
+    public readonly subscription: SubscriptionModels.Subscription;
+    public readonly _isSlot: boolean;
```

Comment:
I created public get methods in cases where it may be used outside of the class.  I think AppSettingsNodes specifically made never need that functionality and I could probably get rid of it, but the wizard does get the subscription and site from the SiteNode itself.

I plan to make all the variable name changes in another PR (where private variables are prefixed with an underscore)